completion/tests/test_utilities.py

```diff
@@ -0,0 +1,39 @@
+from mock import Mock
+
+from django.test import TestCase
+from opaque_keys.edx.keys import CourseKey
+
+from ..exceptions import UnavailableCompletionData
+from ..utilities import get_key_to_last_completed_course_block
+from ..test_utils import UserFactory, CompletionSetUpMixin, submit_completions_for_testing
+
+
+class TestCompletionUtilities(CompletionSetUpMixin, TestCase):
+
+    COMPLETION_SWITCH_ENABLED = True
+    
+    def setUp(self):
+        super(TestCompletionUtilities, self).setUp()
+        self.user = UserFactory.create()
+        self.course_key = CourseKey.from_string("course-v1:edX+MOOC101+2049_T2")
+        self.block_keys = [
+            self.course_key.make_usage_key('video', unicode(number))
+            for number in range(5)
+        ]
+        submit_completions_for_testing(self.user, self.course_key, self.block_keys)
+
+    def test_can_get_key_to_last_completed_course_block(self):
+        last_block_key = get_key_to_last_completed_course_block(
+            self.user,
+            self.course_id
```

Comment:
How would this pass test if only `self.course_key` is defined but `self.course_id` is not?